src/modules/ssh_docker/entity/docker_container.ts

```diff
@@ -0,0 +1,103 @@
+import { MountConfig } from 'dockerode';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { cloudId } from '../../../services/cloud-id';
+import { SshCredentials } from '../../ssh_accounts/entity';
+
+export enum dockerContainerStates {
+  created = 'created',
+  restarting = 'restarting',
+  running = 'running',
+  removing = 'removing',
+  paused = 'paused',
+  exited = 'exited',
+  dead = 'dead',
+}
+
+@Entity()
+export class DockerContainer {
+  /**
+   * @private
+   * Internal ID field for container
+   */
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @cloudId
+  @Column({
+    type: 'character varying',
+    nullable: false,
+  })
+  @ManyToOne(() => SshCredentials, { nullable: false })
+  @JoinColumn([
+    {
+      name: 'server_name',
+      referencedColumnName: 'name',
+    },
+  ])
+  serverName: string;
+
+  @cloudId
+  @Column({ nullable: true })
+  containerId?: string;
+
+  @Column({
+    type: 'varchar',
+    nullable: true,
+  })
+  name?: string;
+
+  @Column()
+  image: string;
+
+  @Column({ nullable: true, type: 'jsonb' })
```

Comment:
What about (based on [this](https://stackoverflow.com/a/57611955) )

```suggestion
  @Column({ nullable: true, type: 'varchar', array: true })
```

I also wonder if we can have a constraint requiring the `varname=value` syntax, but definitely not a blocker.

Reply:
I think this one is doable, but I'm sure it was throwing errors in some cases (when the field is an `array` of `json objects`).

Reply:
I changed it to array, but deferred adding the `Check` as an improvement follow-up.